react-whib/src/components/Sidebar.js

```diff
@@ -1,17 +1,30 @@
 import '../css/Sidebar.css';
-import { useState } from 'react';
-import { Link, NavLink } from 'react-router-dom';
+import { useContext, useEffect, useState } from 'react';
+import { Link, NavLink, useNavigate } from 'react-router-dom';
 import homeIcon from '../img/home-128.png';
 import mapIcon from '../img/map-128.png';
 import profileIcon from '../img/profile-128.png';
+import { UserContext } from '../contexts/AuthContext';
 
 function Sidebar(){
+    const {token, setToken} = useContext(UserContext)
     const [isActive, setActive] = useState(false)
+    const navigate = useNavigate()
 
     function toggleOpen(){
         setActive(!isActive)
     }
 
+    useEffect(()=>{
+
+    }, [token])
+
+    function logout(){
+        setToken(null);
+        localStorage.setItem('access_token', null);
```

Comment:
Hey, I checked it and it doesn't work as intended. You have to use .removeItem instead of setItem because right now it sets it to "access_token" = "null"  ( i mean string null not null null).

Oh and also remove token type with this token.